tests/test_parsers.py

```diff
@@ -79,23 +79,50 @@ def test_gelfparser_parse_partially_invalid_file(caplog):
                 # Another assumed valid gelf
                 '{"short_message": {"username": "This seems valid too."}}\n',
             ]
+        )        
+
+        with caplog.at_level(logging.DEBUG):
+            file.seek(0)
+            parser = GELFParser()
+            events = list(parser.parse(file))
+
+        assert(
+            (
+            "Input event '{ This is not valid json and raises json.decoder.JSONDecodeError\n' "
+            "is not a valid JSON string! It will be ignored."
+            )
+            in caplog.text
         )
+
+        assert(
+            "Input event '{}\n' doesn't comply with GELF format! It will be ignored."
+            in caplog.text
+        )
+
+        assert(
+            "Input event '[]\n' is not a valid JSON string! It will be ignored."
+            in caplog.text
+        )
```

Comment:
With a logging level set as DEBUG, you should see extra messages that needs to be tested (`Error was...`)

Reply:
My test structure is incorrect actually. A proper test would be logging.ERROR with `"Input event..."` messages and logging.DEBUG with `"Raised error..."`. WDYT?